Mathlib/Algebra/Homology/DerivedCategory/Ext/Basic.lean

```diff
@@ -422,6 +423,27 @@ noncomputable def extFunctor (n : ℕ) : Cᵒᵖ ⥤ C ⥤ AddCommGrp.{w} where
     apply Ext.comp_assoc
     all_goals omega
 
+instance (X : C) (n : ℕ) : (extFunctorObj X n).Additive where
+  map_add {_ _ _ _} := by
+    apply AddCommGrp.hom_ext
+    simp [Ext.mk₀_add]
+
+instance (n : ℕ) : (extFunctor (C := C) n).Additive where
+  map_add {_ _ _ _} := by
+    ext
+    simp [Ext.mk₀_add]
+
+/-- `Ext` commutes with finite coproducts. -/
+noncomputable def Ext.coprodIso (X : C) {ι : Type*} [Finite ι] (Y : ι → C) (n : ℕ) :
+    AddCommGrp.of (Ext X (∐ Y) n) ≅ ∐ (fun i => AddCommGrp.of (Ext X (Y i) n)) :=
+  PreservesCoproduct.iso (extFunctorObj X n) Y
+
+/-- `Ext` commutes with finite products. -/
+-- TODO: Prove that `Ext` commutes with arbitrary products.
```

Comment:
I do not think this is true in general. It seems to me it is somewhat related to the question whether infinite products are exact in `C`, which cannot be true in arbitrary abelian categories.
```suggestion
```